app/main.py

```diff
@@ -2,0 +1,34 @@
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 heals: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = heals
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
```

Comment:
Code Style: You should use interpolation instead of concatenation. In this case, instead of `f"{{Name: {self.name}, "`, you should use `f"{{Name: {self.name}, "`. Refer to point 3 in the checklist.